src/test/kotlin/com/openlattice/data/ids/PostgresEntityKeyIdServiceTest.kt

```diff
@@ -147,9 +147,9 @@ class PostgresEntityKeyIdServiceTest : TestServer() {
     @Test
     fun testMassiveInsertAndReadBack() {
         logger.info("Testing insertion and massive readback")
-        val smallBatchSize = 8192
-        val medBatchSize = 32768
-        val largeBatchSize = 128000
+        val smallBatchSize = 128
+        val medBatchSize = 512
+        val largeBatchSize = 2048
```

Comment:
It didn't trigger the error at values of any size and I had to update the build.gradle to prevent OOMs. I left them in, because they did catch some logic errors in entity key id generation separate from the deadlock (mainly around duplicate key assignment) and you didn't need ridiculous batch sizes for that.